spec/features/users/user_authenticates_github_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+# As a user
+# When I visit /dashboard
+# Then I should see a link that is styled like a button that says "Connect to Github"
+# And when I click on "Connect to Github"
+# Then I should go through the OAuth process
+# And I should be redirected to /dashboard
+# And I should see all of the content from the previous Github stories (repos, followers, and following)
+
```

Comment:
We will want to remove this user story later (probably after testing for the content that should appear on the dashboard)